tequilapi/endpoints/current_ip.go

```diff
@@ -0,0 +1,35 @@
+package endpoints
+
+import (
+	"github.com/julienschmidt/httprouter"
+	"github.com/mysterium/node/tequilapi/utils"
+	"net/http"
+)
+
+type currentIPEndpoint struct {
+	ipResolver ipResolver
```

Comment:
All `ipifyClient` could be this dependency. And finally it could renamed `ipifyClient` -> `IPResolver`

Reply:
Not quite sure I understood the suggestion - are you suggesting to refactor `ipify.Client` interface with all its implementations into functions (instead of a classes), i.e.:
`func ipify.NewClient (ipify.Client, error)` -> `func ipify.NewIPResolver (IPResolver, error)` ?

Reply:
No, I suggest to inject full ipifyClient here `ipResolver ipify.Client`